profiles/pantheon_decoupled_profile/composer.json

```diff
@@ -15,8 +15,7 @@
   ],
   "require": {
     "drupal/pantheon_decoupled": "^1.0@alpha",
-    "drupal/gin": "^3.0@beta",
-    "drupal/pathauto": "^1.9"
```

Comment:
Not introduced by you, but I don't think this dependency belongs here in the first place.